examples/gke_github/main.tf

```diff
@@ -0,0 +1,138 @@
+terraform {
+  required_version = ">= 0.12.10"
+
+  required_providers {
+    github = {
+      source  = "hashicorp/github"
```

Comment:
Should be changed to the new provider location.
https://registry.terraform.io/providers/integrations/github/latest

Reply:
Fixed. https://github.com/fluxcd/terraform-provider-flux/pull/112/commits/edf45674341ff251635b0ec238f122cc2c81409e